nulib/meadow
# Update Authoritex to v1.0.0

# Summary 
Update Authoritex to v1.0.0

# Specific Changes in this PR
- Bump Authoritex version
- Add LCGFT and Homosaurus
- Fix bug in code list seed task

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test

1. Run `mix meadow.seed`
2. Start the Meadow web server
3. Navigate to a work and edit
4. Make sure LCGFT and Homosaurus are in the authority list and return data

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
